src/main/java/net/tiagofar78/prisonescape/game/prisonbuilding/SoundDetector.java

```diff
@@ -124,13 +130,46 @@ private void deleteFromWorld() {
         location.getBlock().setType(Material.AIR);
     }
 
+    private int calculateValue() {
+        return isDetectingSound() ? 1 : 0;
+    }
+
     private void updateValue() {
-        int value = isDetectingSound() ? MAX_VALUE : MIN_VALUE;
+        updateScoreboards(calculateValue());
+    }
+
+    private double noiseFunction(int x, int value) {
+        double intensity = 0.1;
+        return value + (_random.nextDouble() * 2 - 1) * intensity;
+    }
+
+    private void updateScoreboards(int value) {
```

Comment:
Would maybe rename this one updateNoiseMeters or updateAudioLevelMeters just to be clearer, so it doesnt create confusion with the players scores.

But that's up to you